VERSION

```diff
@@ -0,0 +1 @@
+v15.1
```

Comment:
IMHO - bad idea to commit that into git 
file should be generated on build (when there is right environment) and then appear in package
in the way that all relevant tools (like PyCharm) will see it